oauth2client/client.py

```diff
@@ -37,6 +37,7 @@
 from oauth2client import clientsecrets
 from oauth2client import transport
 from oauth2client import util
+from oauth2client.pkce import code_verifier, code_challenge
```

Comment:
Yes, it is.

>Use imports for packages and modules only.

[Google Python Style Guide](https://google.github.io/styleguide/pyguide.html?showone=Imports#Imports)